metadata-ws/pom.xml

```diff
@@ -84,6 +84,15 @@
             <version>${cucumber.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Would a dependency like this help to include these new ones?

```xml
<dependency>
  <groupId>org.springframework.boot</groupId>
  <artifactId>spring-boot-starter-security</artifactId>
</dependency>
```

Reply:
Are you suggesting to include `spring-boot-starter-security` artifact?

Reply:
Yes, that may allow us to drop the more specific dependencies, because the starter will deal with them.

Reply:
I guess even spring-boot-starter-security is not needed here, we can include spring security oauth2 without specifying a version.

Reply:
@cyenyxe your inputs are welcome

Reply:
Cristina asked to include starter security to remove the specific dependency of oauth2, but even without adding starter security you should be able to add oauth2 without version spring boot starter parent takes care of the version.